drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -137,6 +140,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                         $fbkTab.addClass('hidden');
                         $zoneTab.removeClass('hidden');
                         self.scrollToTop();
+                        $zoneTab.find('input:first').select();
 
                         $(this).one('click', function loadThirdTab(e) {
```

Comment:
I believe there are preventDefault calls you can remove in this file (and perhaps others) now that the a-tags have been converted to buttons.